File: back/KaemeWebApi/Dominio/Models/Fornecedor.cs
namespace Dominio.Models
{
    public partial class Fornecedor
    {
        public int? Id { get; set; }
        public string Razao_Social { get; set; }
        public string? Logradouro { get; set; }
        public string? Uf { get; set; }
        public string? Cnpj { get; set; }
        public string? Instagram { get; set; }
        public string? Email { get; set; }
        public string? Telefone { get; set; }
        public decimal? Min_Pedido_Atacado { get; set; }
        public decimal? Perc_Desc_A_Vista { get; set; }
        public int? Tipo_Frete_Id { get; set; }
        public DateTime? Data_Criacao { get; set; }
        public int? Criado_Por { get; set; }


        // Campos auxiliares
        public string? Tipo_Frete { get; set; }
    }
}

File: front/kaemeweb/src/app/modules/shared/models/fornecedor.ts
export interface Fornecedor {
  id: number | null;
  razao_Social: string;
  logradouro: string | null;
  uf: string | null;
  cnpj: string | null;
  instagram: string | null;
  email: string | null;
  telefone: string | null;
  min_Pedido_Atacado: number | null;
  perc_Desc_A_Vista: number | null;
  tipo_Frete_Id: number | null;
  data_Criacao: Date | null;
  criado_Por: number | null;

  // Campos auxiliares
  tipo_Frete?: string | null;
}
